test/lint/run-lint-format-strings.py

```diff
@@ -15,6 +15,8 @@
 FALSE_POSITIVES = [
     ("src/clientversion.cpp", "strprintf(_(COPYRIGHT_HOLDERS), COPYRIGHT_HOLDERS_SUBSTITUTION)"),
     ("src/test/translation_tests.cpp", "strprintf(format, arg)"),
+    ("src/test/util_string_tests.cpp", 'tfm::format(std::string{"%*s"}, "hi", "hi")'),
```

Comment:
What's the added value for extending this linter as well?

Reply:
Was getting CI failures, so added these exceptions. I considered extending the linter implementation to handle them correctly, but part of the raison d'étre of `ConstevalFormatString` is getting further towards being able to remove the linter in the first place.

Reply:
Ah like these new ones you mean?
```
[20:48:24.756] src/test/util_string_tests.cpp: Expected 0 argument(s) after format string but found 1 argument(s): tfm::format(ConstevalFormatString<2>{"%*s"}, "hi", "hi")
[20:48:24.756] src/test/util_string_tests.cpp: Expected 0 argument(s) after format string but found 1 argument(s): tfm::format(ConstevalFormatString<2>{"%.*s"}, "hi", "hi")
```

Reply:
Exactly :D

Fixed in latest push.

@maflcko I realize you were concerned about touching this linter in https://github.com/bitcoin/bitcoin/pull/30933#discussion_r1768048796, do you have an alternate approach suggestion?